### DISCUSSION TITLE
Outdated Fedora copr

### DISCUSSION THREAD

**Discussion by EWBennett** [Ulauncher/Ulauncher#1412] [Packaging]
Hi, not sure if this is the right place for this but the [fedora dev package](https://copr.fedorainfracloud.org/coprs/troycurtisjr/ulauncher/) is very out of date - is it possible for this to be updated, or are there instructions for how to build from source I've missed? Thanks!

**Comment by friday** [comment]
I don't use fedora, so I don't know why you'd use that when there's a regular package?

Anyway @troycurtisjr is the copr package owner, but he might not have the time/interest to maintain it.

**Comment by troycurtisjr** [comment]
@friday is right, you can use the package out of the main repo for the latest stable build (currently 5.15.7) so the copr package is currently unnecessary. It was used when there was a major change in the way we were building the RPM as a development aid. However, now that the package is in the main repos it isn't needed (as mentioned in the Copr readme).

Eventually, the 6.x versions will be built first in that Copr before moving to the main repos, and I've been intending to get that going at some point, but for now packages are only available for the stable builds. Were you looking for 6.x releases?

**Reply by EWBennett** [reply]
Yeah, as it's listed on the website under the dev release v6 beta alongside the AUR repo and the v6 .deb file I assumed that was what was intended to be in there. I don't mind building it from source myself if there's a way I can do that on fedora?

**Reply by friday** [reply]
We should update the website then. Thanks for making us aware.

**Reply by friday** [reply]
(but I can't answer your question unfortunately)

**Reply by friday** [reply]
Updated the website, so now it says: ![image](https://github.com/user-attachments/assets/3920ae5b-e706-4904-a324-110511ebca05)

https://github.com/Ulauncher/ulauncher.io/pull/22

Hopefully v6 stable isn't that far away. I have been trying to wrap up development for v6 for a long time now. There's only a few big things left and documentation, but I have very little time for it.
